Comanche055/CONTRACT_AND_APPROVALS.agc

```diff
@@ -14,60 +14,59 @@
 # thanks to both.  The images (with suitable reduction in storage size and
 # consequent reduction in image quality as well) are available online at
 # www.ibiblio.org/apollo.  If for some reason you find that the images are
-# illegible, contact me at info@sandroid.org about getting access to the 
+# illegible, contact me at info@sandroid.org about getting access to the
 # (much) higher-quality images which Paul actually created.
 #
 # Notations on the hardcopy document read, in part:
 #
 #	Assemble revision 055 of AGC program Comanche by NASA
-#	2021113-051.  10:28 APR. 1, 1969  
+#	2021113-051.  10:28 APR. 1, 1969
 
 # Page 1
 
-#************************************************************************
-#									*
-#		THIS AGC PROGRAM SHALL ALSO BE REFERRED TO AS:		*
-#									*
-#									*
-#				COLOSSUS 2A				*
-#									*
-#									*
-#	THIS PROGRAM IS INTENDED FOR USE IN THE CM AS SPECIFIED		*
-#	IN REPORT R-577.  THIS PROGRAM WAS PREPARED UNDER DSR		*
-#	PROJECT 55-23870, SPONSORED BY THE MANNED SPACECRAFT		*
-#	CENTER OF THE NATIONAL AERONAUTICS AND SPACE			*
-#	ADMINISTRATION THROUGH CONTRACT NAS 9-4065 WITH THE		*
-#	INSTRUMENTATION LABORATORY, MASSACHUSETTS INSTITUTE OF		*
-#	TECHNOLOGY, CAMBRIDGE, MASS.					*
-#									*
-#************************************************************************
+# *************************************************************************
+# *									  *
+# *	     THIS AGC PROGRAM SHALL ALSO BE REFERRED TO AS:		  *
+# *									  *
+# *									  *
+# *		            COLOSSUS 2A				          *
```

Comment:
Patch should fix this, also added a space for the first line in this box which I never noticed on both checks of this file.